src/Microsoft.DotNet.Build.Tasks/NormalizeAssemblyName.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using System;
+
+namespace Microsoft.DotNet.Build.Tasks
+{
+    public class NormalizeAssemblyName : Task
```

Comment:
It should inherit from BuildTask so that when we cross compile it for netfx it has the AssemblyResolver magic.

Reply:
although we don't need this one here as I am not adding any new dependencies but I'll do it anyway.